troubleshooting/troubleshooting-push-notifications-are-currently-initializing.md

```diff
@@ -0,0 +1,54 @@
+---
+title: Push notifications are currently initializing
+description: Push notifications are currently initializing error recieved when trying to register a device for push notifications on Android or iOS.
+type: troubleshooting
+page_title: Push notifications are currently initializing
+slug: troubleshooting-push-notifications-are-currently-initializing
+tags: push-notifications
+teampulseid:
+ticketid: 1073013, 907763
+pitsid:
+
+---
+
+## Environment
+<table>
+  <tr>
+    <td>Service</td>
+    <td>Notifications (Backend Services)</td>
+  </tr>
+  <tr>
+    <td>Feature</td>
+    <td>Push Notifications</td>
+  </tr>
+</table>
+
+## Description
+
+Trying to register the device for push notifications using *everlive.push.register()* returns an error message: "Push notifications are currently initializing".
+
+## Error Message
+"Push notifications are currently initializing"
+
+## Cause\Possible Cause(s)
+This error is returned from Telerik Platform JavaScript SDK and means that the initial call to *everlive.push.register()* method has not yet finished, but it was called again by the app.
+
+## Solution
+There could be several reasons why the initial call to *everlive.push.register()* method has not yet finished before being invoked again:

```

Comment:
We can have only one solution. Here my suggestion is to explain that the native part of the push initialization logic is not finished and something is preventing it. (This can be added in the Cause section, though). 

In the solution we may just specify that the:

- Deploy the application is deployed as an application package on an actual device (do not mention the companion app - which is now called Cordova developer app). A link to the article with deploying may be handy. 
- Solution: For iOS, check that the provisioning profile used to deploy the app is for explicit Apple App ID and features the Push Notifications entitlement.